Subproject2/1. SOVA/.editorconfig

```diff
@@ -0,0 +1,4 @@
+﻿[*.cs]
+
+# CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
+dotnet_diagnostic.CS8632.severity = silent
```

Comment:
This is simply to make the IDE shut up about nullable property something something. Read the comments provided by IDE